cmd/server_test.go

```diff
@@ -1843,6 +1843,8 @@ func (s *TestSuiteCommon) TestGetPartialObjectMisAligned(c *C) {
 	1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,
 	1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,1234567890,
 	1234567890,1234567890,1234567890,123`
+	// ensuring that the buffer contains atleast 13 bytes of data.
+	buffer.WriteString(line[:13])
```

Comment:
Part of erasure index distribution which we used to have can be used here, this was before from what we have moved to today.

```
// randInts - uses Knuth Fisher-Yates shuffle algorithm for generating uniform shuffling.
func randInts(count int) []int {
        rand.Seed(time.Now().UTC().UnixNano()) // Seed with current time.
        ints := make([]int, count)
        for i := 0; i < count; i++ {
                ints[i] = i + 1
        }
        for i := 0; i < count; i++ {
                // Choose index uniformly in [i, count-1]
                r := i + rand.Intn(count-i)
                ints[r], ints[i] = ints[i], ints[r]
        }
        return ints
}
```